pages/admin/vault.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { useState, useEffect } from "react";
 import { useTranslation } from "next-i18next";
 import axios from "axios";
-import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+// import { serverSideTranslations } from "next-i18next/serverSideTranslations";
```

Comment:
Why this change ? Doesn't really matter since this page is not really used. Honestly if you want to make changes here you might as well just remove the page